mindmeld/components/_elasticsearch_helpers.py

```diff
@@ -272,26 +341,28 @@ def load_index(
         # create the progess bar with docs count
         pbar = tqdm(total=docs_count)
 
-        for okay, result in streaming_bulk(
-            es_client,
-            docs,
-            index=scoped_index_name,
-            doc_type=doc_type,
-            chunk_size=50,
-            raise_on_error=False,
+        es_version_7 = is_es_version_7(es_client)
+        for okay, result in version_compatible_streaming_bulk(
+            es_client, docs, scoped_index_name, 50, False, DOC_TYPE
         ):
-
             action, result = result.popitem()
-            doc_id = "/%s/%s/%s" % (index_name, doc_type, result["_id"])
+            if es_version_7:
+                doc_id = "/%s/%s" % (index_name, result["_id"])
+            else:
+                doc_id = "/%s/%s/%s" % (index_name, doc_type, result["_id"])
+
             # process the information from ES whether the document has been
             # successfully indexed
             if not okay:
                 logger.error("Failed to %s document %s: %r", action, doc_id, result)
             else:
                 count += 1
             pbar.update(1)
+
         # close the progress bar and flush all output
         pbar.close()
+        es_client.indices.flush(index=scoped_index_name)
+        es_client.indices.refresh(index=scoped_index_name)
```

Comment:
@mhuang2173 Could you review this?

Reply:
i wonder do we need both? from their release note documentation looks like we would only need `refresh()` to make the indexed docs searchable?

Reply:
the tests were failing without doing both, and in the latest release refresh is no longer part of flush: https://www.elastic.co/guide/en/elasticsearch/reference/current/breaking-changes-7.0.html#flush-force-merge-no-longer-refresh

Reply:
@arushir Can we just use the `refresh` here and remove `flush`? Looks like bulk upload does an implicit `flush`

Reply:
Interesting, yes I'll try that.

Reply:
Updated to use only refresh